docs/_sections/installation.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Installation
+---
+
+## Installation Guide
+Releases of DVRescue can be downloaded for all supported systems from its page on the [MediaArea website](https://mediaarea.net/DVRescue).
+
+It also is possible to download specific snapshots and daily builds from MediaArea via their download page [here](https://old.mediaarea.net/download/snapshots/binary/dvrescue/).
```

Comment:
Recommended:

It also is possible to download specific snapshots and daily builds from MediaArea via their [download page](https://old.mediaarea.net/download/snapshots/binary/dvrescue/).

Reply:
Don't want to point basic users there for the time being - there are links to the dailies deeper in the docs for people getting more down in the weeds. Can get to snapshots from within MediaArea as well